docs/python/3.opdracht1.md

```diff
@@ -3,82 +3,106 @@ sidebar_position: 3
 ---
 
 # Opdracht 1: Rekenmachine
-Je zit op je opdracht in een nieuw project. Voor dit project moet een rekenmachine worden gebouwd. Dit moet je doen door gebruik te maken van Behaviour-driven development BDD. Vanuit de business heb je de volgende vier requirements gekregen:
 
-Requirements:
+Je zit op je opdracht in een nieuw project. Voor dit project moet een rekenmachine worden gebouwd. Dit moet je doen door
+gebruik te maken van Behaviour-driven development BDD. Vanuit de business heb je de volgende vier requirements gekregen:
 
-- Optellen: de rekenmachine moet een methode bieden om twee of meer getallen bij elkaar op te tellen, met ondersteuning voor decimalen en negatieve waarden.
+### Requirements:
 
-- Aftrekken: de rekenmachine moet een methode bevatten om het ene getal van het andere af te trekken, inclusief decimalen en negatieve waarden.
+- **Optellen:** de rekenmachine moet een methode bieden om twee of meer getallen bij elkaar op te tellen, met
+  ondersteuning voor decimalen en negatieve waarden.
 
-- Vermenigvuldiging: de rekenmachine moet een methode bieden om twee of meer getallen met elkaar te vermenigvuldigen, waarbij decimalen en negatieve waarden worden ondersteund.
+- **Aftrekken:** de rekenmachine moet een methode bevatten om het ene getal van het andere af te trekken, inclusief
+  decimalen en negatieve waarden.
 
-- Delen: de rekenmachine moet een methode bevatten om het ene getal door het andere te delen, met ondersteuning voor decimalen en het omgaan met deling door nul-fouten.
+- **Vermenigvuldiging:** de rekenmachine moet een methode bieden om twee of meer getallen met elkaar te
+  vermenigvuldigen, waarbij decimalen en negatieve waarden worden ondersteund.
+
+- **Delen:** de rekenmachine moet een methode bevatten om het ene getal door het andere te delen, met ondersteuning voor
+  decimalen en het omgaan met deling door nul-fouten.
 
 ``` !!!!! IMPORTANT NOTE: Voordat we gaan beginnen, straks gaan we gebruiken maken van ChatGPT, controleer zelf het antwoordt dat wordt teruggegeven door ChatGPT! De gegeneerde code is namelijk niet altijd 100% correct maar geeft je een zetje in de goede richting. Dit zorgt voor een kritische blik op het gene wat je laat genereren. ```
 
-##  1.1 Cucumber Feature file maken.
+## 1.1 Cucumber Feature file maken.
 
-Een feature file beschrijft een scenario in begrijpelijke mensentaal. In die feature file wordt het gewenste gedrag beschreven welke stappen er genomen dienen te worden om tot een bepaald resultaat te komen.
+Een feature file beschrijft een scenario in begrijpelijke mensentaal. In die feature file wordt het gewenste gedrag
+beschreven welke stappen er genomen dienen te worden om tot een bepaald resultaat te komen.
 
 Een voorbeeld van een feature file ziet er uit als volgt:
-```
+
+```yml
 Scenario: Scenarionaam
-    Given I have one pizza
-    And Cut the pizza in 8 slices
-    When I eat 2 slices
-    Then The result is 6 slices left
+  Given I have one pizza
```

Comment:
indentation is nodig hier, valt namelijk onder dat scenario

Reply:
is er ook nog